modules/tinymce/src/core/main/ts/init/Render.ts

```diff
@@ -100,19 +99,21 @@ const loadIcons = (scriptLoader: ScriptLoader, editor: Editor, suffix: string) =
   });
 };
 
-const loadPlugins = (editor: Editor, settings: EditorSettings, suffix: string) => {
-  if (Type.isArray(settings.plugins)) {
-    settings.plugins = settings.plugins.join(' ');
+const loadPlugins = (editor: Editor, suffix: string) => {
+  // These lines should be changed to a getParam and some type of setParam once such an API is available.
```

Comment:
Is there a Jira for this? Otherwise we'll probably forget...

Reply:
Looks like this line should be removed, as settings should only ever be a string by this point from what I can see.

Reply:
So take away the check for array?

Reply:
I thought I found an example in the docs last week that uses `plugins` as an array but can't find it now... Would make sense to remove the comment and array check though, yes

Reply:
Nvm the first half, just saw Lee's next reply.